Source/d_main.c

```diff
@@ -2016,6 +2016,9 @@ void D_DoomMain(void)
 
   putchar('\n');     // killough 3/6/98: add a newline, by popular demand :)
 
+  // process .deh files specified on the command line with -deh or -bex.
+  D_ProcessDehCommandLine();
```

Comment:
Note that we still have a slightly different order than in Choco:
https://github.com/chocolate-doom/chocolate-doom/blob/3524a67dd3c7d48a172c83d3ae170a7176fb3cfa/src/doom/d_main.c#L1576-L1581

Reply:
Yes, but we do not distinguish between DEHACKED lumps in IWADs or PWADs and we have one auto-load feature more than Choco (i.e. the two entries in the menu). I guess it's more important to do what Pr+ does, as always. 😉